test/python/circuit/library/test_evolution_gate.py

```diff
@@ -113,6 +113,29 @@ def test_suzuki_trotter_manual(self):
 
         self.assertEqual(evo_gate.definition.decompose(), expected)
 
+    @idata(
+        [
+            [X + Y, 0.5, 1],
+            [X, 0.238, 2],
+        ]
+    )
```

Comment:
There's actually no need to use `idata` here; you can just use regular `data` and remove the outer list:
```python
@data(
    (X + Y, 0.5, 1),
    (X, 0.238, 2),
)
@unwrap
```

Reply:
Now uses `data` instead of `idata`.